apps/extension/src/ui/domains/Account/AccountExportModal.tsx

```diff
@@ -19,17 +19,29 @@ import { PasswordUnlock, usePasswordUnlock } from "./PasswordUnlock"
 const EXPORTABLE_ORIGINS: AccountType[] = ["SEED", "JSON", "DERIVED"]
 
 const useAccountExportModalProvider = () => {
-  const { account } = useSelectedAccount()
-  const { isOpen, open, close } = useOpenClose()
+  const [_account, setAccount] = useState<AccountJsonAny>()
+
+  const { account: selectedAccount } = useSelectedAccount()
+  const { isOpen, open: innerOpen, close } = useOpenClose()
+
+  const open = useCallback(
+    (account?: AccountJsonAny) => {
+      setAccount(account)
+      innerOpen()
+    },
+    [innerOpen]
+  )
 
   useEffect(() => {
     close()
-  }, [account, close])
+  }, [selectedAccount, close])
 
-  const canExportAccount = useMemo(
-    () => account?.origin && EXPORTABLE_ORIGINS.includes(account?.origin),
-    [account]
-  )
+  const account = _account ?? selectedAccount
```

Comment:
@0xKheops I had to add this logic to all of the account options.

Previously, the user couldn't only run these against the selected account.  
But now they can run them against any account without selecting it via the Manage Accounts settings page.

Reply:
ok, i think we can sunset these providers soon anyway. 
They were there because those actions needed to take place on the globally selected account.
But now that we finally have a manage accounts screen, IMO the portfolio's context menu should be nuked.

@chidg @alecdwm wdyt, should we remove it right away ?